### DISCUSSION TITLE
Failure to start Kernel due to Modules not installed

### DISCUSSION THREAD

**Discussion by nazifa02** [giswqs/geohey#15] [Q&A] [answered]
Hello everyone, has anyone faced this issue **'kernel failed to start'** while running code in Visual Studio? 

![Screenshot 2024-03-27 220031](https://github.com/giswqs/geohey/assets/116415085/3b78f261-f849-41d7-ad6a-be56fad6a9ec)

I've installed prompt_toolkit but getting the same error. 
```pip install prompt_toolkit```

**Comment by giswqs** [comment]
Did you create a new conda env to install packages?

**Reply by nazifa02** [reply]
@giswqs No, I used the existing conda environment and installed prompt_toolkit there

**Reply by giswqs** [reply, marked as answer]
so has the issue been resolved? If not, create a new conda env to install packages

**Reply by nazifa02** [reply]
It worked for new conda environment. Thank you, Dr. Wu.
